src/main/resources/application.properties

```diff
@@ -1 +1,4 @@
-
+spring.jpa.hibernate.ddl-auto=none
```

Comment:
If there's ddl-auto=none - so where do you put all SQL Create statements?

Either use different ddl-auto or introduce a database upgrade tasks (like flyway or liquidbase)